pyproject.toml

```diff
@@ -18,6 +18,16 @@ select = [
     "B"
 ]
 
+[tool.codespell]
```

Comment:
Please move this `tool.codespell` table to the head of the file, to preserve alphabetical ordering (by table name). This makes it a little easier to navigate the file as it gets bigger.